src/pages/Details.js

```diff
@@ -1,11 +1,66 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 export default function Details() {
   return (
     <div className="details">
       <div className="content">
-        <div className="img-details">
-          <img src="/img/cuate.svg" />
+        <div className="info-questions">
+          <div className="img-details">
+            <img src="/img/cuate.svg" />
+          </div>
+          <div className="info-text">
+            <p>
+              A) NPS (Net Promoter Score): Avaliado com valores numéricos entre
+              1 e 10. Para o intervalo numérico entre 1 a 6 classificamos como
+              Avaliação Detratora, entre os valores 7 e 8 como avaliação Neutra
+              e entre os valores 9 e 10 como uma avaliação Promotora.
+            </p>
+            <div className="info-options">
+              <p>1</p>
+              <p>2</p>
+              <p>3</p>
+              <p>4</p>
+              <p>5</p>
+            </div>
+            <div className="info-options">
+              <p>6</p>
+              <p>7</p>
+              <p>8</p>
+              <p>9</p>
+              <p>10</p>
+            </div>
```

Comment:
É uma lista, usa tag de lista
```suggestion
            <ul className="nps-list">
              <li>1</li>
              <li>2</li>
              <li>3</li>
              <li>4</li>
              <li>5</li>
              <li>6</li>
              <li>7</li>
              <li>8</li>
              <li>9</li>
              <li>10</li>
            </ul>
```